docs/source/examples/audio-sentence.rst

```diff
@@ -126,7 +126,7 @@ can get one by index.
         )
     ]
 
-In ``sentence`` attribute of :ref:`features:Sentence`, there is a list of :ref:`features:Word`.
+In ``sentence`` attribute of :ref:`features:Sentence`, there is a list of Word.
```

Comment:
Add "Word" under "docs/source/features.rst/Supported Label Types/Sentence/", and keep the cross-reference.